README.md

```diff
@@ -969,6 +969,13 @@ The information required to connect to Kafka will be automatically set for you b
 
 Before you begin fixing issues or adding new features to the Kogito Operator, see [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD) and [Kogito Operator architecture](docs/ARCHITECTURE.MD).
 
+### Requirements
+
+- [Docker](https://www.docker.com/)
+- [Operator Courier](https://github.com/operator-framework/operator-courier) is used to build, validate and push Operator Artifacts
+- [Operator SDK](https://github.com/operator-framework/operator-sdk) V0.16.0 or later
+- [Golint dependency](golang.org/x/lint/golint): go get -u golang.org/x/lint/golint
```

Comment:
I think we should also mention Go 1.13.x requirement, see chapter Kogito Operator requirements

Reply:
Added